ParabolInc/action
# Add grouping grid

TEST
- start a meeting in 2 browsers
- create 20 cards, go to grouping phase

BASIC ACTIONS
- [ ] client A can pick up a card, client B sees it picked up by client A
- [ ] client A can release a card & it animates home for both client A & B
- [ ] client A can drag a card onto another one to form a group, client B sees it
- [ ] client A can repeat the above step quickly
- [ ] client A can ungroup 2 cards from the group quickly and client B sees 2 cards controlled by client A being ungrouped
- [ ]  client A can regroup the cards

MODAL BEHAVIOR
- [ ] client A can click the group & see the modal open
- [ ] client A can ungroup from the modal
- [ ] client B can regroup the cards
- [ ] client A can open the modal that client B just created
- [ ] with the modal open on client A, client B can drag a card into the group. the modal grows to accept the addition
- [ ] client B can remove a card from the modal, the modal shrinks

WINDOW BEHAVIOR
- [ ] client A can resize the window. the cards rearrange themselves
- [ ] client A can select a card on the top & drag & scroll it to the bottom
- [ ] client A can use a browser other than chrome to drag

AUTO GROUP BEHAVIOR
- [ ] client A can click auto group & the cards fall into groups

VOTING
- [ ] client A can progress the meeting to the voting phase
- [ ] client A can expand the modal & vote from the modal
